newIDE/app/src/MainFrame/ElectronMainMenu.js

```diff
@@ -91,9 +92,20 @@ const buildAndSendMenuTemplate = (
     label: i18n._(t`File`),
     submenu: [
       {
-        label: i18n._(t`Create a New Project...`),
-        accelerator: getElectronAccelerator(shortcutMap['CREATE_NEW_PROJECT']),
-        onClickSendEvent: 'main-menu-create',
+        label: i18n._(t`Create`),
+        submenu: [
+          {
+            label: i18n._(t`New empty project...`),
+            accelerator: getElectronAccelerator(
+              shortcutMap['CREATE_NEW_PROJECT']
+            ),
+            onClickSendEvent: 'main-menu-create-blank',
+          },
+          {
+            label: i18n._(t`New project from template...`),
```

Comment:
Just a thought: are the ellipsis' needed?

Reply:
Not really, but I copied the current design which seems to be:

- If this action is the final one, then no ellipsis
- If there will be a following dialog, then ellipsis

I think it makes sense